### PR TITLE
feat: add left right column condition for content

### LINKED ISSUE #169: Add ability to align General Info block text (description) left or right when content align is Horizontal
For example, [Loupe This case study](https://www.sanctuary.computer/loupe-this) has a General Info block titled `1 o’clock, 2 o’clock, sold!`, we want to add an option to align the description to the right, when content align is horizontal. [Link to Contentful case study](https://app.contentful.com/spaces/zb716c7gdesq/entries/4fUUjtfWUSdPs8cckDBhZW)

Screen Shot 2023-01-18 at 1 54 05 PM

Screen Shot 2023-01-18 at 1 55 21 PM

### PR DESCRIPTION
**Pull request by nahbee10**
### Description

- in Contentful, add `horizontalContentColumn` field to Case Study Block - General Info to give option to the editors to change the left-right column alignment for the horizontal content
- add optional basement style classes to the header/body text wrapper divs of `components/BlockGeneralInfo.js` to implement left-right column option from `horizontalContentColumn` field

### Type(s) of changes

- [ ] Bug fix
- [x] New feature
- [x] Update to an existing feature

### Motivation for PR

Closes #169 

### How Has This Been Tested?

- tested with production dataset on local environment

### Applicable screenshots

- new field(horizontalContentColumn) usage on Contentful([Link](https://app.contentful.com/spaces/zb716c7gdesq/entries/4fUUjtfWUSdPs8cckDBhZW))
Screen Shot 2023-01-19 at 8 05 02 AM
Screen Shot 2023-01-19 at 8 05 31 AM

- implemented on component
Screen Shot 2023-01-19 at 8 04 08 AM

### CODE REVIEW

**Comment by mokaymokay** [comment]
doesn't seem to be working? https://sanctu-dot-com-git-feature-align-t-3370e4-sanctucompudevelopers.vercel.app/loupe-this

**Comment by nahbee10** [comment]
> doesn't seem to be working? https://sanctu-dot-com-git-feature-align-t-3370e4-sanctucompudevelopers.vercel.app/loupe-this

@mokaymokay [preview](https://sanctu-dot-kdp06wojo-sanctucompu.vercel.app/loupe-this) is here! It might not work because I pushed it before I changed the field on the Contentful!

**Comment by mokaymokay** [comment]
Noted one change here: https://www.loom.com/share/7216c32453a74f568b3d6cd35659164c

Thanks Nahee!

**Comment by nahbee10** [comment]
> Noted one change here: https://www.loom.com/share/7216c32453a74f568b3d6cd35659164c
> 
> Thanks Nahee!

Thanks for pointing this out @mokaymokay ! Fixed it and [preview is here](https://sanctu-dot-1vpka6efg-sanctucompu.vercel.app/loupe-this). Below is the screenshot of the column after the fix.
Screen Shot 2023-01-19 at 1 49 43 PM

File: components/BlockGeneralInfo.js [hunk 1]
```diff
@@ -9,42 +9,60 @@ import { SimpleFragment } from 'models';
...
+    fields,
+    'horizontalContentColumn',
+    'Left'
+  ).toLowerCase();
   const marginBottom = get(fields, 'marginBottom', 1);
   const marginTop = get(fields, 'marginTop', 1);
 
+  /* frequently used class conditions */
+  const isContentHorizontalRight =
+    contentAlign === 'horizontal' && horizontalContentColumn !== 'left';
```
Comment (mokaymokay) [inline components/BlockGeneralInfo.js:29]: this is a slight nitpick, but i think this is easier to read:

```
const isContentHorizontalRight =
    contentAlign === 'horizontal' && horizontalContentColumn === 'right';
```

**Review by mokaymokay — APPROVED** [review]

**Pull request merged**